src/store/trees/trees.js

```diff
@@ -20,15 +20,28 @@ class ValidationError extends Error {
   }
 }
 
+let configOverrides = {
+  appConfigOverrides: {},
+  builderConfigOverrides: {},
+}
+
 export default {
   modules: {flights},
 
+  appConfigOverrides: {},
+  builderConfigOverrides: {},
+
+  configure(appConfig, builderConfig) {
+    configOverrides.appConfigOverrides = appConfig
+    configOverrides.builderConfigOverrides = builderConfig
```

Comment:
I'm not sure I understand how this would work. Moving it below the module registration would mean that the config overrides are not available when state()  gets called (which I think is during module registration right?)

Reply:
Maybe let's discuss in the catch up